xmtpv3/bindings/wasm/crate/src/lib.rs

```diff
@@ -1,6 +1,28 @@
+use std::{sync::Mutex};
+
+use libxmtp_core::persistence::InMemoryPersistence;
+use libxmtp_core::{client::Client};
 use wasm_bindgen::prelude::*;
 
+static CLIENT_LIST: Mutex<Vec<Client>> = Mutex::new(Vec::new());
+
+#[wasm_bindgen]
+pub fn client_create() -> usize {
+    let mut clients = CLIENT_LIST.lock().unwrap();
+    clients.push(Client::new(Box::new(InMemoryPersistence::new())));
+    clients.len() - 1
+}
+
+#[wasm_bindgen]
+pub fn client_write_to_persistence(client_id: usize, key: String, value: &[u8]) -> Result<(), String> {
+    let mut clients = CLIENT_LIST.lock().unwrap();
+    let client = clients.get_mut(client_id).expect("Client not found");
+    client.write_to_persistence(key, value)
+}
+
```

Comment:
This file doesn't look like it was formatted using rustfmt. Might want to add it to your editor to minimize headaches